tests/quality/config.yaml

```diff
@@ -155,6 +155,8 @@ tests:
       - nvd:cpe
 
   - provider: oracle
+    additional_trigger_globs:
+      - src/vunnel/utils/oval_parser.py
```

Comment:
good call -- I had been pondering adding some analysis of the imports to automatically figure this, but this is much better than the doesn't-exist-hypothetical